tests/rptest/tests/group_membership_test.py

```diff
@@ -88,11 +88,11 @@ def test_list_groups_has_no_duplicates(self):
         kcl.consume(self.topic, n=1, group="g1")
         kcl.consume(self.topic, n=1, group="g2")
         kcl.consume(self.topic, n=1, group="g3")
-        # transfer kafka_internal/group/0 leadership across the nodes to trigger
+        # transfer kafka/__consumer_offsets/0 leadership across the nodes to trigger
```

Comment:
Does this commit have to come bundled with a previous one to keep tests passing?

Reply:
i wanted to keep the commits clean. they should all compile but there may be a times when not all dt tests are passing